_components/tooltips.md

```diff
@@ -0,0 +1,74 @@
+        <li><strong>Redundant content.</strong> Do not use where the information is repetitive or usability is obvious.</li>
+        <li><strong>Sufficient space.</strong> If your UI can fit the content you like to put inside of a tooltip, consider not using a tooltip.</li>
+      </ul>
+      <h4>Usability guidance</h4>
+      <ul class="usa-content-list">
+        <li><strong>Use affordances.</strong> Make it clear that an element can be interacted with when using tooltips.</li>
+        <li><strong>Avoid collisions.</strong> Be careful not introduce conflicting hover or focus events.</li>
+        <li><strong>Use consistently.</strong> If using tooltips in one context, use in all similar contexts.</li>
+        <li><strong>Do not block content.</strong> Use the <code>data-position</code> attribute to prevent the tooltip from covering elements in way that distracts the user.</li>
+      </ul>
+      <h4 class="usa-heading">Accessibility</h4>
+      <ul class="usa-content-list">
+        <li><strong>Use as title attribute.</strong> Tooltips are essentially a progressive enhancement for a title attribute, and will gracefully degrade to it if the component does not intialize.</li>
+        <li><strong>Keyboard accessibility</strong> Tooltips make title attributes keyboard accessible.</li>
+      </ul>
+      <h4 class="usa-heading">Implementation</h4>
+      <ul class="usa-content-list">
+        <li>Any element given the class name <code>usa-tooltip</code> and a <code>title</code> attribute will become a tooltip.</li>
+        <li>Place tooltips on elements that contain little or no child elements</li>
+        <li>Elements or text that show a tooltip when hovered or focused will be prevented wrapping onto a new line and will be given <code>tabindex="0"</code> for keyboard interaction.</li>
+        <li>By default, tooltips will appear on top of the element that triggers them</li>
+        <li>Use the <code>data-position</code> attribute to indicate where tooltip appears in relation to the element that makes it appear:
+          <ul>
+            <li><code>data-position="top"</code>: On top, horizontally centered. If the <code>data-position</code> attribute is omitted, the tooltip will appear on top by default.</li>
+            <li><code>data-position="bottom"</code>: Below, horizontally centered</li>
+            <li><code>data-position="right"</code>: To the right, vertically centered</li>
+            <li><code>data-position="left"</code>: To the left, vertically centered</li>
+          </ul>
+        </li>
+        <li>Tooltips are protected from clipped by the viewport. If clipping is detected, the tooltip is positioned to the opposite side indicated in the <code>data-position</code> attribute. If the tooltip is still clipped, it placed on top of the element, with its width constrained to the width of the triggering element. Only then does the tooltip wrap to multiple lines.</li>
```

Comment:
- `from clipped by the viewport` → `viewport clipping`
- `to the opposite side indicated in the` → `on the opposite side`
- `it placed on` → `it is positioned on`

Reply:
Done.